### ISSUE TITLE
AttributeError when adding Meosif middleware

### ISSUE THREAD

**Issue by hoangphuc05** [Moesif/moesifdjangoexample#16]
After following the instruction here at https://www.moesif.com/wrap/onboard/integration?role=engineering&goal=improve-devex, my django api return error on all api endpoint `'Response' object has no attribute '_headers'`

**Comment by dgilling** [comment]
@hoangphuc05 Would you be able to share the Django Version and Python version? Feel free to email support@moesif.com if that's easier.

**Comment by hoangphuc05** [comment]
Hi,
My current Django version: 3.2.4, Python version: 3.9.6
Moesif version:
```python
moesifapi==1.3.3
moesifdjango==2.0.4
moesifpythonrequest==0.2.0
```

My middleware in case it is useful
```python
MIDDLEWARE = [
    'corsheaders.middleware.CorsMiddleware',
    'django.middleware.security.SecurityMiddleware',
    'django.contrib.sessions.middleware.SessionMiddleware',
    'django.middleware.common.CommonMiddleware',
    'django.middleware.csrf.CsrfViewMiddleware',
    'django.contrib.auth.middleware.AuthenticationMiddleware',
    'django.contrib.messages.middleware.MessageMiddleware',
    'django.middleware.clickjacking.XFrameOptionsMiddleware',
    'moesifdjango.middleware.moesif_middleware'
]
```

moesif setting
```python
def identifyUser(req, res):
    if req.user and req.user.is_authenticated:
        return req.user.username
    else:
        return None


MOESIF_MIDDLEWARE = {
    'APPLICATION_ID': '....',

    'CAPTURE_OUTGOING_REQUESTS': False,  # Set to True to also capture outgoing calls to 3rd parties.

    'IDENTIFY_USER': identifyUser  # Optional hook to link API calls to users
}
```

**Comment by keyur9** [comment]
Hello, @hoangphuc05! Thank you for letting us know about the issue. We've fixed it and released a new version `moesifdjango 2.0.5` to support Response attribute for Django 3.2+. Let us know if still facing any issue when using Django 3.2+. Thank you.

**Comment by hoangphuc05** [comment]
it's working right now for me. Thank you very much!
